cypress/support/step-definitions/decimal-steps.js

```diff
@@ -1,65 +1,70 @@
+  commonDataElementInputPreviewNoIframe().parent().should('not.have.attr', 'disabled');
 });
 
 Then('Decimal component is readOnly', () => {
-  commonDataElementInputPreview().should('have.attr', 'readonly');
-  commonDataElementInputPreview().parent().should('have.attr', 'readonly');
+  commonDataElementInputPreviewNoIframe().should('have.attr', 'readonly');
+  commonDataElementInputPreviewNoIframe().parent().should('have.attr', 'readonly');
 });
 
 Then('Decimal component is not readOnly', () => {
-  commonDataElementInputPreview().should('not.have.attr', 'readonly');
-  commonDataElementInputPreview().parent().should('not.have.attr', 'readonly');
+  commonDataElementInputPreviewNoIframe().should('not.have.attr', 'readonly');
+  commonDataElementInputPreviewNoIframe().parent().should('not.have.attr', 'readonly');
 });
 
-When('label width on preview is {int}', (width) => {
+Then('label width on preview is {int}', (width) => {
+  getDataElementByValueNoIframe('label').should('have.attr', 'width').should('contain', `${width}`);
+});
+
+Then('label width on preview is {int} in IFrame', (width) => {
   label().should('have.attr', 'width').should('contain', `${width}`);
 });
 
-When('inputWidth on preview is {int}', (width) => {
-  commonDataElementInputPreview().parent().should('have.css', 'flex').should('contain', `${width}%`);
+Then('inputWidth on preview is {int}', (width) => {
+  commonDataElementInputPreviewNoIframe().parent().should('have.css', 'flex').should('contain', `${width}%`);
 });
 
 Then('label align on preview is set to {string}', (labelAlign) => {
+  getDataElementByValueNoIframe('label').should('have.css', TEXT_ALIGN, `${labelAlign}`);
+});
+
+Then('label align on preview is set to {string} in IFrame', (labelAlign) => {
   label().should('have.css', TEXT_ALIGN, `${labelAlign}`);
 });
 
+
 When('I set Decimal input to {word}', (labelInput) => {
-  commonDataElementInputPreview().clear().type(labelInput);
+  commonDataElementInputPreviewNoIframe().clear().type(labelInput);
 });
 
 Then('Decimal Input is set to {word}', (labelInput) => {
-  commonDataElementInputPreview().should('have.attr', 'value').should('contain', `${labelInput}`);
+  commonDataElementInputPreviewNoIframe().should('have.attr', 'value').should('contain', `${labelInput}`);
 });
 
-Then('Decimal label is not set to {word}', (labelInput) => {
-  commonDataElementInputPreview().should('have.attr', 'value').should('not.contain', `${labelInput}`);
+Then('Decimal input is not set to {word}', (labelInput) => {
+  commonDataElementInputPreviewNoIframe().should('have.attr', 'value').should('not.contain', `${labelInput}`);
 });
 
-When('I set input precision slider to {int}', (precision) => {
-  setSlidebar(inputPrecisionSlider(), precision);
-});
-
-Then('input precision slider is set to {int}', (precision) => {
-  inputPrecisionSlider().should('have.attr', 'value').should('contain', `${precision}`);
+Then('label is inline', () => {
```

Comment:
It looks like a reuseable step - move it to common file

Reply:
moved to `common-steps.js` file